bridge/client/app/_services/data.service.ts

```diff
@@ -93,10 +95,27 @@ export class DataService {
     return this._projectName.asObservable();
   }
 
+  get hasUnreadUniformRegistrationLogs(): Observable<boolean> {
+    return this._hasUnreadUniformRegistrationLogs.asObservable();
+  }
+
+  public setHasUnreadUniformRegistrationLogs(status: boolean): void {
+    this._hasUnreadUniformRegistrationLogs.next(status);
+  }
+
   public setProjectName(projectName: string): void {
     this._projectName.next(projectName);
   }
 
+  public setUniformDate(integrationId: string, lastSeen?: string) {
+    this._uniformDates[integrationId] = lastSeen || new Date().toISOString();
```

Comment:
As we then use probably Date in the components pass `lastSeen` as Date and use then `lastSeen.toISOString()`.

Reply:
`lastSeen` comes from `uniformRegistrationLog.time`, not from `lastSeen` of the component, therefore it is type string

Reply:
👍